site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -239,22 +235,23 @@ scenarios:
   `team-a-cq` can admit Workloads with resources adding up to `12+9=21` CPUs and
   `48+36=84Gi` of memory.
 - If ClusterQueue `team-b-cq` has pending Workloads and the ClusterQueue
-  `team-a-cq` has all its `min` quota used, Kueue will admit Workloads in
+  `team-a-cq` has all its `nominalQuota` quota used, Kueue will admit Workloads in
   ClusterQueue `team-b-cq` before admitting any new Workloads in `team-a-cq`.
-  Therefore, Kueue ensures the `min` quota for `team-b-cq` is met.
+  Therefore, Kueue ensures the `nominalQuota` quota for `team-b-cq` is met.
 
-**Note**: Kueue [does not support preemption](https://github.com/kubernetes-sigs/kueue/issues/83).
-Workloads already admitted by Kueue will _not_ be stopped to make space for new Workloads.
+**Note**:
+Kueue supports preemption and Workloads already admitted by Kueue may be 
```

Comment:
```suggestion
Kueue supports [preemption](#preemption) and Workloads already admitted by Kueue may be 
```

Reply:
Either add the link or just remove the section given that preemption is described below.

Reply:
Removed.